### ISSUE TITLE
Transfer learning for general weights

### ISSUE THREAD

**Issue by arielkantorovich** [Deci-AI/super-gradients#1013]
This is a more general question and less issue I will be happy if you can help me.
My question is how I make transfer learning on my weights train, I will be more concrete if I train from scratch yolo-nas small on VisDrone2019 dataset, to train from scratch if I understand right I don't use the flag pretrained_weights.
Now my goal is to take the best results.pth and transfer these weights to a new dataset how can I implement this?
pretrained_weights='path to relevant weights' ?
Thank you for a great project I will be happy to get help :)

**Comment by Louis-Dupont** [comment]
Hi @arielkantorovich ,
Exactly, `pretrained_weights` is to load the pretrained weights directly from the model zoo.
We have some tutorials on how to do this, you can have a look at the [quickstart](https://docs.deci.ai/super-gradients/documentation/source/QuickstartBasicToolkit.html), it shows an example of how to load a model you trained yourself.
For more information about checkpoints, you can also check this [documentation page](https://docs.deci.ai/super-gradients/documentation/source/Checkpoints.html).
We also have tutorials on other subjects which might help you going through your task, feel free to check it out

Let me know if anything is not clear or if you need more information :)
